docs/install.md

```diff
@@ -14,147 +14,194 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Installation
+# Recommended Installation Method
 
-There are 4 ways to use Triton Model Analyzer:
+<br>
 
-## Building the Dockerfile
+## Triton SDK Container with Docker Launch Mode
 
-The recommended way to use Model Analyzer is by building the Model Analyzer's
-docker yourself. This installation method uses `--triton-launch-mode=local` by
-default, as all the dependencies will be available. First, clone the Model
-Analyzer's git repository, then build the docker image.
+---
+
+The recommended way to install Model Analyzer is from the Triton SDK docker
+container available on the [NVIDIA GPU Cloud
+Catalog](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver).<br><br>
+**1. Pull the SDK container:**
 
 ```
-$ git clone https://github.com/triton-inference-server/model_analyzer.git -b <rXX.YY>
+docker pull nvcr.io/nvidia/tritonserver:22.05-py3-sdk
```

Comment:
Is it possible to avoid hardcoding 22.05 here? Else Misha will have to update every release. Or at least only put it in one place and explain how to pick the latest?

Could do similar to the description in the install file, and use an env variable?

Step 1: Pick release version
(explain rxxyy)
`export RELEASE=<r22.05>`

Step 2: Pull the SDK container
`docker pull nvcr.io/nvidia/tritonserver:${RELEASE}-py3-sdk`

...

Reply:
Done